lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -997,6 +998,20 @@ SubExprInfo ExprEmitter::visitComb(ArraySliceOp op) {
   return {Unary, arrayPrec.signedness};
 }
 
+// Syntax from: section 5.11 "Array literals".
+SubExprInfo ExprEmitter::visitComb(ArrayCreateOp op) {
+  os << '{';
+  SubExprInfo exprInfo =
+      /* dummy init value. */ {Symbol, SubExprSignResult::IsUnsigned};
+  llvm::interleaveComma(op.inputs(), os, [&](Value operand) {
+    os << "'{";
+    exprInfo = emitSubExpr(operand, LowestPrecedence);
+    os << "}";
+  });
+  os << '}';
+  return {Unary, exprInfo.signedness};
```

Comment:
This should always return an unsigned value, because array literals are unsigned, or they don't care about sign.   That allows you to sink the definition of exprInfo into the closure.